### PR TITLE
zh: add IAM ENV for import and export in backup

### PR DESCRIPTION
**Pull request by shuijing198799**
### What is changed, added or deleted? (Required)
add IAM ENV for import and export in backup

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [ ] master (the latest development version, including v1.1 changes for now)

### CODE REVIEW

File: zh/backup-to-s3.md [hunk 1]
```diff
@@ -16,35 +16,82 @@ Ad-hoc 全量备份通过创建一个自定义的 `Backup` custom resource (CR)
...
 目前兼容 S3 的存储中，Ceph 和 Amazon S3 经测试可正常工作。下文对 Ceph 和 Amazon S3 这两种存储的使用进行描述。本文档提供如下备份示例。示例假设对部署在 Kubernetes `test1` 这个 namespace 中的 TiDB 集群 `demo1` 进行数据备份，下面是具体操作过程。
 
+### AWS 账号的三种权限授予方式
+
+如果使用 Amazon S3 来备份恢复集群，可以使用三种权限授予方式授予权限，参考[使用 BR 工具备份 AWS 上的 TiDB 集群](backup-to-aws-s3.md#aws-账号权限授予的三种方式)，使用 Ceph 作为后端存储测试备份恢复时，是通过 AccessKey 和 SecretKey 模式授权。
+
 ### Ad-hoc 全量备份环境准备
 
-1. 下载文件 [backup-rbac.yaml](https://github.com/pingcap/tidb-operator/blob/master/manifests/backup/backup-rbac.yaml)，并执行以下命令在 `test1` 这个 namespace 中创建备份需要的 RBAC 相关资源：
+参考[Ad-hoc 全量备份环境准备](backup-to-aws-s3.md#ad-hoc-全量备份环境准备)
```
Comment (ran-huang) [inline zh/backup-to-s3.md]:
```suggestion
参考 [Ad-hoc 全量备份环境准备](backup-to-aws-s3.md#ad-hoc-全量备份环境准备)
```

Comment (DanielZhangQD) [reply zh/backup-to-s3.md]: The file name for br has changed.

File: zh/backup-to-s3.md [hunk 2]
```diff
@@ -16,35 +16,82 @@ Ad-hoc 全量备份通过创建一个自定义的 `Backup` custom resource (CR)
...
+
+### 备份数据到兼容 S3 的存储
+
++ 创建 `Backup` CR，通过 accessKey 和 secretKey 授权的方式将数据备份到 Amazon S3：
```
Comment (DanielZhangQD) [inline zh/backup-to-s3.md]: Please keep the same format for all of the `accessKey` `secretKey`.

File: zh/restore-from-s3.md [hunk 3]
```diff
@@ -10,29 +10,54 @@ category: how-to
...
-1. 下载文件 [`backup-rbac.yaml`](https://github.com/pingcap/tidb-operator/blob/master/manifests/backup/backup-rbac.yaml)，并在 `test2` 这个 namespace 中创建恢复所需的 RBAC 资源，所需命令如下：
+参考[环境准备](restore-form-aws-s3.md#环境准备)
 
-    {{< copyable "shell-regular" >}}
+## 将指定备份数据恢复到 TiDB 集群
 
-    ```shell
-    kubectl apply -f backup-rbac.yaml -n test2
-    ```
+1. 创建 restore customer resource (CR)，将制定备份数据恢复至 TiDB 集群
```
Comment (DanielZhangQD) [inline zh/restore-from-s3.md]: ALL of the `restore` -> `Restore`

File: zh/backup-to-s3.md (hunk 1 above)
Comment (shuijing198799) [reply zh/backup-to-s3.md]: done

File: zh/backup-to-s3.md (hunk 2 above)
Comment (shuijing198799) [reply zh/backup-to-s3.md]: done

File: zh/restore-from-s3.md (hunk 3 above)
Comment (shuijing198799) [reply zh/restore-from-s3.md]: done

File: zh/restore-from-s3.md [hunk 4]
```diff
@@ -10,29 +10,54 @@ category: how-to
...
+## AWS 账号的三种权限授予方式
+
+如果使用 Amazon S3 来备份恢复集群，可以使用三种权限授予方式授予权限，参考[使用 BR 工具备份 AWS 上的 TiDB 集群](backup-to-aws-s3-using-br.md#aws-账号权限授予的三种方式)，使用 Ceph 作为后端存储测试备份恢复时，是通过 AccessKey 和 SecretKey 模式授权。
+
 以下示例将兼容 S3 的存储（指定路径）上的备份数据恢复到 TiDB 集群。
 
 ## 环境准备
 
-1. 下载文件 [`backup-rbac.yaml`](https://github.com/pingcap/tidb-operator/blob/master/manifests/backup/backup-rbac.yaml)，并在 `test2` 这个 namespace 中创建恢复所需的 RBAC 资源，所需命令如下：
+参考[环境准备](restore-form-aws-s3.md#环境准备)
```
Comment (DanielZhangQD) [inline zh/restore-from-s3.md]: file name changed.

File: zh/backup-to-s3.md [hunk 5]
```diff
@@ -16,35 +16,82 @@ Ad-hoc 全量备份通过创建一个自定义的 `Backup` custom resource (CR)
...
+        # region: us-east-1
+        # storageClass: STANDARD_IA
+        # acl: private
+        # endpoint:
+      storageClassName: local-storage
+      storageSize: 10Gi
     ```
 
-3. 创建 `backup-demo1-tidb-secret` secret。该 secret 存放用于访问 TiDB 集群的 root 账号和密钥。
++ 创建 `Backup` CR，通过 AccessKey 和 SecretKey 授权的方式将数据备份到 Ceph：
```
Comment (ran-huang) [inline zh/backup-to-s3.md]: @shuijing198799 Is it `Ceph` or `Amazon S3`?

File: zh/restore-from-s3.md [hunk 6]
```diff
@@ -10,29 +10,54 @@ category: how-to
...
-    {{< copyable "shell-regular" >}}
+## 将指定备份数据恢复到 TiDB 集群
 
-    ```shell
-    kubectl apply -f backup-rbac.yaml -n test2
-    ```
+1. 创建 Restore customer resource (CR)，将制定备份数据恢复至 TiDB 集群
 
-2. 创建 `restore-demo2-tidb-secret` secret，该 secret 存放用来访问 TiDB 集群的 root 账号和密钥：
++ 创建 Restore custom resource (CR)，通过 AccessKey 和 SecretKey 授权的方式将指定的备份数据恢复至 TiDB 集群：
```
Comment (ran-huang) [inline zh/restore-from-s3.md]: @shuijing198799 麻烦看一下，L27 和 L60 是否重复了？

Comment (ran-huang) [reply zh/restore-from-s3.md]: Fixed in #57

**Pull request merged**